tests/factories/transcript.py

```diff
@@ -0,0 +1,28 @@
+from factory import Sequence
+from factory.alchemy import SQLAlchemyModelFactory
+
+from via.models import Transcript
+
+
+class TranscriptFactory(SQLAlchemyModelFactory):
+    class Meta:
+        model = Transcript
+
+    video_id = Sequence(lambda n: f"video_id_{n}")
+    transcript = [
+        {
+            "text": "[Music]",
+            "start": 0.0,
+            "duration": 7.52,
+        },
+        {
+            "text": "how many of you remember the first time",
+            "start": 5.6,
+            "duration": 4.72,
+        },
+        {
+            "text": "you saw a playstation 1 game if you were",
+            "start": 7.52,
+            "duration": 4.72,
+        },
+    ]
```

Comment:
This factory is already useful as-is but in future we'll probably be adding more fields to transcripts (e.g. maybe a language code) so it'll become even more useful to have a factory for generating realistic instances